client/human/HumanClientFSM.cpp

```diff
@@ -578,6 +578,29 @@ boost::statechart::result MPLobby::react(const ChatHistory& msg) {
     return discard_event();
 }
 
+boost::statechart::result MPLobby::react(const PlayerStatus& msg) {
+    TraceLogger(FSM) << "(HumanClientFSM) PlayerStatus.";
+    // ToDo: show it in player ready status
+
+    return discard_event();
+}
+boost::statechart::result MPLobby::react(const SaveGameComplete& msg) {
```

Comment:
in general, prefer spaces between `}` and next function definition. rest of this file doesn't appear to be an exception.

Reply:
Fixed.